IDGPredict/FacetImage.h

```diff
@@ -22,14 +22,16 @@ class FacetImage
 	
 	int OffsetY() const { return _offsetY; }
 	
-	void Set(size_t width, size_t height, double value)
+	void Set(size_t width, size_t height, size_t spectralTerms, double value)
 	{
 		_width = width;
 		_height = height;
-		_data.assign(width * height, value);
+    _data.resize(spectralTerms);
```

Comment:
Please use consistent tabs or spaces (holds for entire PR).

Reply:
Indeed that was a bit messy; most IDGPredict files still used tabs because I started those in a different project. I've ran them all through `expand` now so that they use spaces consistently.